roles/serviceassurance/tasks/component_qdr.yml

```diff
@@ -33,14 +33,13 @@
             saslMechanisms: ANONYMOUS
             sslProfile: openstack
         interRouterListeners:
-          - authenticatePeer: true
-            expose: true
+          - expose: true
```

Comment:
what does expose mean here? I couldn't find it in the qdrouter documentation?

Reply:
https://access.redhat.com/documentation/en-us/red_hat_amq/7.5/html/deploying_amq_interconnect_on_openshift/connecting-clients-router-network-router-ocp#exposing-ports-router-ocp